packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -31,6 +31,87 @@ describe('HttpValidator', () => {
       });
     });
 
+    describe('deprecated keyword validation', () => {
+      const resource = {
+        id: "abc",
+        method: "get",
+        path: "/test",
+        responses: [
+          {
+            code: "200",
+            contents: [
+              {
+                mediaType: "application/json",
+                schema: {
+                  type: "object",
+                  properties: {
+                    prop: {
+                      type: "string",
+                      example: "abc"
+                    }
+                  },
+                  $schema: "http://json-schema.org/draft-04/schema#"
```

Comment:
You can make this response way shorted, since this is not what we're testing. Provide something basic and save some lines.